week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -7,3 +7,31 @@ Assign each of them to a separate variable
 
 */
 

```

Comment:
Good.   Half of England can't spell "vegetables", so you're doing well!  `shoppingList` - capitalisation.